Workbooks/Workloads/Create log alert/Create log alert.workbook

```diff
@@ -78,7 +78,7 @@
             "label": "Alert name",
             "type": 1,
             "isRequired": true,
-            "query": "Resources\r\n| take 1\r\n| project id = extract(@'/subscriptions/.+/resourcegroups/.+/providers/microsoft.operationalinsights/workspaces/.+/alertname/(.+)/threshold/.+/template/.+', 1, tolower('{WorkspaceAndThrehsold}'))\r\n| project value = id, label = id, selected = true",
+            "query": "Resources\r\n| take 1\r\n| project id = extract(@'(?i)/subscriptions/.+/resourcegroups/.+/providers/microsoft.operationalinsights/workspaces/.+/alertname/(.+)/threshold/.+/template/.+', 1, '{WorkspaceAndThrehsold}')\r\n| project value = id, label = id, selected = true",

```

Comment:
Good find with (?i). I need to go back and fix up all the tolowers in my templates.